src/glwidget.cpp

```diff
@@ -33,6 +33,7 @@
 #include <QtEvents>
 #include "post_guard.h"
 
+#include <QPainter>
```

Comment:
This should be inside the `#include "pre_gaurd.h"` ... `#include "post_gaurd.h"` pair...